src/main/java/com/hivemq/extension/sdk/api/services/cluster/ClusterDiscoveryCallback.java

```diff
@@ -33,9 +33,11 @@
 public interface ClusterDiscoveryCallback {
 
     /**
-     * This method is called once by HiveMQ when this callback is added via {@link ClusterService#addDiscoveryCallback(ClusterDiscoveryCallback)}.
+     * This method is called once by HiveMQ when this callback is added via
+     * {@link ClusterService#addDiscoveryCallback(ClusterDiscoveryCallback)}.
      * <p>
-     * It can be used to register this HiveMQ instance with some kind of central registry or to save it to a database or
+     * It can be used to register this HiveMQ instance with some kind of central registry or to save it to a database
+     * or
```

Comment:
weird

Reply:
I've rolled back this change (I anyway had to rebase the backports due to the version bump)...